apps/public/src/app/(map)/locatie/kies/page.module.css

```diff
@@ -0,0 +1,58 @@
+.page {
+  grid-template-rows: auto 1fr auto;
+  height: 100vh;
+}
+
+.assetList {
+  display: none;
+  grid-row-start: 2;
+  position: relative;
+
+  @media screen and (min-width: 68rem) {
+    display: block;
+  }
+}
+
+.toggleAssetList {
+  display: block;
+}
+
+.hideButtonDesktop {
+  @media screen and (min-width: 68rem) {
+    display: none;
+  }
+}
+
+.hideButtonMobile {
+  display: none;
+
+  @media screen and (min-width: 68rem) {
+    display: block;
+  }
+}
+
+.map {
+  grid-row-start: 2;
+  position: relative;
+
+  @media screen and (min-width: 68rem) {
+    grid-row: 1 / -1;
+  }
+}
+
+.button {
+  bottom: 0;
+  padding-block: var(--ams-space-sm);
+  padding-inline: var(--ams-space-sm);
+  position: absolute;
+  z-index: 702;
+}
+
+.submitButton {
+  composes: button;
```

Comment:
This isn't really necessary, you can define the rules here and `composes: submitButton` in `toggleButton`

Reply:
Sure.